models/marts/ledger_current_state/contract_data_current.yml

```diff
@@ -30,51 +30,120 @@ models:
         
       - name: contract_key_type
         description: '{{ doc("contract_key_type") }}'
+        tests:
+          - not_null:
+              config:
+                where: closed_at > current_timestamp - interval 2 day
 
       - name: contract_durability
         description: '{{ doc("contract_durability") }}'
+        tests:
+          - not_null:
+              config:
+                where: closed_at > current_timestamp - interval 2 day
 
       - name: asset_code
         description: '{{ doc("asset_code") }}'
+        tests:
+          - not_null:
+              config:
+                where: closed_at > current_timestamp - interval 2 day
 
       - name: asset_issuer
         description: '{{ doc("asset_issuer") }}'
+        tests:
+          - not_null:
+              config:
+                where: closed_at > current_timestamp - interval 2 day
 
       - name: asset_type
         description: '{{ doc("asset_type") }}'
+        tests:
+          - not_null:
+              config:
+                where: closed_at > current_timestamp - interval 2 day
 
       - name: balance_holder
         description: '{{ doc("balance_holder") }}'
+        tests:
+          - not_null:
+              config:
+                where: closed_at > current_timestamp - interval 2 day
```

Comment:
Does this test actually pass? I think there are blank balance_holders (maybe with empty string?) in the table